plugins/inputs/t128_tank/t128_tank.go

```diff
@@ -0,0 +1,84 @@
+package t128_tank
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+var cliPath = "/opt/128technology/bin/tank-cli"
+var topic = "events"
+
+var sampleConfig = `
+[[inputs.t128_tank]]
+## Required. A name for the collector which will be used as the measurement name of the produced data.
+# collector_name = "event_collector"
+
+## Required.
+# log_name = "event_collector"
+
+## Required.  
+# index_file = ""
+
+## Required.
+# topic = "events"
```

Comment:
We'll also want to provide the IP and port for the server as we do have multiple running now.